jupyter-pytorch-full/tests/test_rock.py

```diff
@@ -7,16 +7,13 @@
 import pytest
 import subprocess
 import yaml
-from pytest_operator.plugin import OpsTest
 
 @pytest.mark.abort_on_fail
-def test_rock(ops_test: OpsTest):
+def test_rock():
     """Test rock."""
     rock = yaml.safe_load(Path("rockcraft.yaml").read_text())
-    name = rock["name"]
+    rock_image = rock["name"]
     rock_version = rock["version"]
```

Comment:
I think you could use chisme abstractions instead of doing this manually as we 're doing [here](https://github.com/canonical/kserve-rocks/blob/main/controller/tests/test_rock.py#L13-L15).

Reply:
done